index.html

```diff
@@ -4399,7 +4399,7 @@ <h5>Conclusion</h5>
 <li><a data-ucr-role="tag" href="">accessibility-potential</a>
 A native implementation would be better able to incorporate a semantically meaningful representation of a map
 into its accessibility tree than current implementations,
-which depend on the accessibility features of more general native elements in conjunction with [[ARIA]].
+which depend on the accessibility features of more general native elements in conjunction with [[WAI-ARIA]].
```

Comment:
```suggestion
which depend on the accessibility features of more general native elements in conjunction with [[[wai-aria-1.1]]].
```

Without the number it links to version 1, and 1.1 is already fully stable (W3C Recommendation).

Reply:
That's only the attribution:

> _Accessible Rich Internet Applications (WAI-ARIA) 1.0. James Craig; Michael Cooper et al. W3C. 20 March 2014. W3C Recommendation. URL: https://www.w3.org/TR/wai-aria/_

however the link would still take you to the latest recommendation. But I accept your changes.